spec/rails_helper.rb

```diff
@@ -54,4 +92,35 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  config.include Devise::Test::ControllerHelpers, type: :controller
+  config.include FactoryGirl::Syntax::Methods
+  config.include ApplicationHelper, type: :view
+  config.include Warden::Test::Helpers, type: :feature
+
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.before(:each) do |example|
```

Comment:
Wondering whether it wouldn't be nicer to use RSpec's before filters, too. https://relishapp.com/rspec/rspec-core/docs/hooks/filters

`config.before(:example, :clean) { ActiveFedora::Cleaner.clean! }`, etc... ? Thoughts?